src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_REPEAT = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0;i < NUMBER_OF_REPEAT;i++) {
```

Comment:
Please, format your code.

Reply:
You can use `Ctrl + Alt + L` or `⌘⌥L` (depending on your OS) to automatically format your code while you are using Intellij Idea. [Here](https://resources.jetbrains.com/storage/products/intellij-idea/docs/IntelliJIDEA_ReferenceCard.pdf) are a lot of other shortcuts.